### ISSUE TITLE
Enable some choices to be public

### ISSUE THREAD

**Issue by wbreeze** [wbreeze/socelect#29]
- Add a public flag to a choice.
- Enable it to be checked or unchecked when editing the choice.
- Have the flag be initially unchecked.
- Display the show links for choices checked public, which are still open, on the landing page.
- Show them in order of closing time, such that choices closing sooner are at the top.

**Comment by wbreeze** [comment]
As I start implementing this I'm imagining the junk that will be published. Two options come immediatly to mind:

- Make it an admin only function that publishes things I want to publish myself
- Make it moderated, with a publication request flag and a second flag, set by a moderator, to actually make it public. This of course requires a moderator interface and access controls. And then some sort of moderator notification that will get all of the spam, a mailer, and etc.

In other words, it becomes much more complicated.

I'm going to press on and deal with the moderation when and if it's actually needed.
